.github/workflows/release.yml

```
@@ -0,0 +1,153 @@
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: [build-exe]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          path: artifacts
+      - name: Source tarball
+        run: git archive --format=tar.gz --prefix=career-ops-${GITHUB_REF_NAME#v}/ -o artifacts/career-ops-${GITHUB_REF_NAME}.tar.gz HEAD
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:49: Double quote to prevent globbing and word splitting

(shellcheck)